src/parse/parse.service.ts

```diff
@@ -71,4 +80,14 @@ export class ParseService {
             countOfPages: countOfPages,
         }
     }
+
+    async fetchSerial(url: string): Promise<MediaResponse> {
```

Comment:
we can use fetchMedia. Don't need to create one more function

Reply:
+